js/store.js

```diff
@@ -43,8 +45,8 @@ export class TodoStore extends EventTarget {
 			'@id': uuid(),
 			'@domain': `${this.id}.todomvc.m-ld.org`,
 			genesis: isNew,
-			io: {uri: `http://${window.location.hostname}:3001`}
-		}).then(async meld => {
+			io: {uri: `http://${window.location.hostname}:3001`},
+		}, new TSeqText('title')).then(async meld => {
```

Comment:
This is the magic word to make the `title` field collaborative text rather than just a string. See

https://edge.js.m-ld.org/classes/tseqtext.html